sdk/lib/dmilib/dmilib.h

```diff
@@ -29,8 +31,16 @@ enum _ID_STRINGS
     ID_STRINGS_MAX,
 };
 
+#ifdef __cplusplus
+extern "C" {
+#endif
 VOID
 ParseSMBiosTables(
     _In_reads_bytes_(TableSize) PVOID SMBiosTables,
     _In_ ULONG TableSize,
     _Inout_updates_(ID_STRINGS_MAX) PCHAR * Strings);
+#ifdef __cplusplus
```

Comment:
Add a separating newline before the ifdef.

Reply:
Nitpicks... :smirk: